src/swoop/api/models/__init__.py

```diff
@@ -295,9 +295,33 @@ class Events(BaseModel):
     pass
 
 
+class PayloadDetails(BaseModel):
+    payload_uuid: str | None = None
+
+
 class PayloadList(BaseModel):
-    pass
+    payloads: list[PayloadDetails]
+    links: list[Link]
+
+
+class Action(BaseModel):
+    action_uuid: str | None = None
+    action_type: str | None = None
+    action_name: str | None = None
+    handler_name: str | None = None
+    parent_uuid: str | None = None
+    created_at: datetime | None = None
+    priority: int | None = None
+    payload_uuid: str | None = None
```

Comment:
Why not reuse the job `StatusInfo` model for this? The idea of an "action" is an internal abstraction we probably shouldn't be exposing to API users.

It seems like it might makes sense to standardize the `to_status_info` function in `routers/jobs.py` (https://github.com/Element84/swoop/blob/main/src/swoop/api/routers/jobs.py#L40) as a method on the `StatusInfo` model, maybe with a name like `from_action_record` or something?